kube/overlays/stable/.env

```diff
@@ -56,6 +56,11 @@ JOB_MAIN_CONTAINER_CPU_LIMIT=
 JOB_MAIN_CONTAINER_MEMORY_REQUEST=
 JOB_MAIN_CONTAINER_MEMORY_LIMIT=
 
+NORMALIZATION_JOB_MAIN_CONTAINER_MEMORY_LIMIT=
+NORMALIZATION_JOB_MAIN_CONTAINER_MEMORY_REQUEST=
+NORMALIZATION_JOB_MAIN_CONTAINER_CPU_LIMIT=
+NORMALIZATION_JOB_MAIN_CONTAINER_CPU_REQUEST=
```

Comment:
Should these also be added to `docker-compose.yml`?

Reply:
Done!